digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs

```diff
@@ -18,18 +19,28 @@ public class ModelDefinitionInterface : DigitalTwinInterfaceClient
 
         private const string getModelDefinitionCommandName = "getModelDefinition";
 
+        private const string EnvironmentalSensorInterfaceFileName = "EnvironmentalSensor.interface.json";
+        private const string EnvironmentalSensorInterfaceFileAssemblyName = "EnvironmentalSensorSample";
+
         public ModelDefinitionInterface(string interfaceInstanceName)
             : base(ModelDefinitionInterfaceId, interfaceInstanceName)
         {
-            environmentalSensorModelDefinition = File.ReadAllText("../../../EnvironmentalSensor.interface.json");
+            var assembly = Assembly.GetExecutingAssembly();
+            var assemblyResourceName = EnvironmentalSensorInterfaceFileAssemblyName + "." + EnvironmentalSensorInterfaceFileName;
+
+            using (Stream stream = assembly.GetManifestResourceStream(assemblyResourceName))
```

Comment:
Consider using the new syntax:

using Stream stream = assembly.GetManifestResourceStream(assemblyResourceName);
using StreamReader reader = new StreamReader(stream);

No need for code blocks anymore. When the variables go out of scope, Dispose will be called.

Reply:
Cool, I didn't know that was an option!